templates/terraform/pre_delete/storage_hmac_key.go.erb

```diff
@@ -0,0 +1,26 @@
+getUrl, err := replaceVars(d, config, "{{StorageBasePath}}projects/{{project}}/hmacKeys/{{access_id}}")
+if err != nil {
+	return err
+}
+
+getRes, err := sendRequest(config, "GET", project, getUrl, nil)
```

Comment:
should we also handle "DELETED" state here?

Reply:
No need- we'll only try to delete if we think the resource exists.